tempesta_fw/Makefile

```diff
@@ -22,6 +22,13 @@ ifdef DEBUG
 	EXTRA_CFLAGS += -DDEBUG=$(DEBUG)
 endif
 
+MBEDTLS = $(obj)/mbedtls
+MBEDTLS_CFLAGS = -I$(MBEDTLS)/include -I$(MBEDTLS)/mbedtls/include \
+	-include $(MBEDTLS)/lk.h -DMBEDTLS_CONFIG_FILE='"$(MBEDTLS)/config.h"'
+export MBEDTLS MBEDTLS_CFLAGS
+
+EXTRA_CFLAGS += $(MBEDTLS_CFLAGS)
```

Comment:
As it's explained at the below, this is not mbedtls, rather it's our own, very specific, fork of the library. So all the variables and functions should be named as `TTLS` (Tempesta TLS) instead of `MBEDTLS`.